relayer/chainTracker/errors.go

```diff
@@ -0,0 +1,16 @@
+package chaintracker
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+var ( // Consumer Side Errors
+	InvalidConfigErrorBlocksToSave  = sdkerrors.New("Invalid blocks to save", 1, "blocks to save wasn't defined in config")
+	InvalidConfigBlockTime          = sdkerrors.New("Invalid average block time", 2, "average block time wasn't defined in config")
```

Comment:
Lets create a more unique errors, 1 digit might overlap with other errors we can go 6 digit errors